kube/Cargo.toml

```diff
@@ -21,6 +21,7 @@ rustls-tls = ["kube-client/rustls-tls", "client"]
 
 # alternative features
 openssl-tls = ["kube-client/openssl-tls", "client"]
+aws-lc-rs = ["kube-client?/aws-lc-rs"]
```

Comment:
I think this is a nice way of doing it, yeah.

Do you mind adding a `--group-features=rustls-tls,aws-lc-rs` to https://github.com/kube-rs/kube/blob/main/justfile#L55 - our feature fuzzing is pretty heavy as it stands